lib.php

```diff
@@ -505,6 +508,7 @@ public function submission_handler($eventdata) {
         $author = $moduleobject->get_author($eventdata['userid'], $eventdata['relateduserid'], $cm, $eventdata['objectid']);
         $groupid = $moduleobject->get_groupid($eventdata['objectid']);
         $submitter = new plagiarism_turnitinsim_user($eventdata['userid']);
+        // remove this
```

Comment:
?

Reply:
Had some debug lines there, seems I removed the debug but forgot to remove the note.